### ISSUE TITLE
Website structure setup - framework, libs, wallet connect

### ISSUE THREAD

**Issue by EvgeniiaVak** [DeXter-on-Radix/website#6]

**Comment by fliebenberg** [comment]
Happy. Will you set up a basic app for us that we can then clone and contribute to Evgeniaa? Anyone who have not yet, I suggest you have a look at the medium article I wrote about connecting AlphaDEX to a React app. https://medium.com/alphadex-xrd/connecting-alphadex-to-your-react-js-app-d8822e99cdcd The article not only explains how to include the SDK, but also explains how to access the data. You can read more about the available variables and functions of the SDK here: https://www.npmjs.com/package/alphadex-sdk-js

**Comment by EvgeniiaVak** [comment]
Yeah, that's a large issue, I probably should have done multiple. But it's possible to have multiple assignees to an issue. I suggest:
*  I take the framework tickbox (there is already a pr - #7 about it)
* maybe @SmashingBumpkin would want to take the data steam from AlphaDEX example
* @fliebenberg would you like to tackle the wallet connection?

(btw I'm not sure how separate the wallet connection and AlphaDEX data are, I haven't looked deeply into them yet)
* then I will be able to wire up the charts I did previously with already live (test network?) data

And at least another dev, or everyone reviews the pull requests.

I guess testing will be a challenge 😆  does AlphaDEX provide something like a testing simulation of trading activity?

**Comment by fliebenberg** [comment]
Happy to take the wallet connection or the Alphadex data stream. I have done both before, so @SmahingBumpkin, you choose the one you want and I will take the other one.

For testing, everything on Alphadex is currently effectively testing as it is all on the RCNet Babylon testnet. So I could create a couple of "Test" pairs on Alphadex if we need to. For testing we can set up a couple of test scenarios with with known results that we can then test again. For the first few issues though, the testing should be very simple.
Alphadex does not provide a simulation unfortunately, so the best it can do is to provide "Test" pairs on the Radix network.

**Comment by SmashingBumpkin** [comment]
I think the Alphadex data stream is my preference.

Just to clarify the general flow of data:

1. We will be getting exchange data via the rest api (pairs, orders, history etc)
2. That data is then used to allow a user to generate a transaction that gets signed using the radix wallet
3. The transaction will then be sent directly to the ledger, or it will go via the API to the ledger?
4.  Will we be displaying open orders that the user has submitted but have yet to be filled? Will these come via the API?

**Comment by EvgeniiaVak** [comment]
@SmashingBumpkin for this first coding issue we don't need to implement all MVP (we kinda don't have the exact list of the MVP, I think designers should lead in this and we will have wireframes for that) in the meantime we need only a tiny thing to get started, for example, ohlc data for a trading pair, so that we could wire up the candlestick chart - just to have some components working together.

rest API? is websocket possible?

**Comment by SmashingBumpkin** [comment]
Of course, i was just interested in how everything was gonna tie together.

And yep websocket, my mistake

**Comment by EvgeniiaVak** [comment]
> Will we be displaying open orders that the user has submitted but have yet to be filled? Will these come via the API?

Everything will come via the API, our app is just a nice visual interface for people to interact with, it will not store any data for long and will not have much "business logic" so to speak.

**Comment by fliebenberg** [comment]
> > Will we be displaying open orders that the user has submitted but have yet to be filled? Will these come via the API?
> 
> Everything will come via the API, our app is just a nice visual interface for people to interact with, it will not store any data for long and will not have much "business logic" so to speak.

User orders (both open, completed and cancelled will need to be pulled from the API. It is unfortunately not part of the live feed from the SDK as it is user specific, so we will have to constantly (say every 10 seconds) ping the API to check for new orders.
